test/powershell/Modules/Microsoft.PowerShell.Management/Restart-Computer.Tests.ps1

```diff
@@ -30,12 +29,18 @@ try
         }
 
         It "Should support -computer parameter" {
+            if ( ! $IsWindows ) {
+                return
+            }
```

Comment:
Please use `It "Should support -computer parameter" -Skip:(!$IsWindows) {`

Reply:
done for all.